eslint_js.yaml

```diff
@@ -0,0 +1,79 @@
+parserOptions:
+  ecmaVersion: 9
+  sourceType: module
+
+env:
+  es6: true
+
+rules:
+  valid-typeof: 2
```

Comment:
Question: Are any of the rules changed here? Or are they all the same? I'm not manually reviewing that they are the same, but I just wanted to make sure I can assume they are.

Reply:
None of these rules are changed, essentially this file is just renamed as `eslint-base.yaml`.

For TS I did need to disable a couple of rules that are switched on for JS, but which don't really work with TS or are better handled by `@typescript-eslint` rules.